src/controllers/QueryController.ts

```diff
@@ -116,7 +116,7 @@ export class QueryController extends RootComponent {
    * @param element
    * @param options
    */
-  constructor(element: HTMLElement, public options: ISearchInterfaceOptions, public usageAnalytics, public searchInterface) {
+  constructor(element: HTMLElement, public options: ISearchInterfaceOptions, public usageAnalytics, public searchInterface: SearchInterface) {
```

Comment:
public usageAnalytics is now useless since we get the one from the SearchInterface
Could I remove it I'm afraid it would be breaking...

Reply:
Yeah, I think you could mark it as deprecated param in the documentation. 

But better to leave it there since it is documented publicly.